app/packages/core/src/components/Actions/ActionsRow.tsx

```diff
@@ -162,7 +162,9 @@ const Tag = ({
   const [available, setAvailable] = useState(true);
   const labels = useRecoilValue(fos.selectedLabelIds);
   const samples = useRecoilValue(fos.selectedSamples);
-  const readOnly = useRecoilValue(fos.readOnly);
+  const canTag = useRecoilValue(fos.canTagSamples);
```

Comment:
should we rename `canTagSamples` to `canTag` or `canTagSamplesOrLabels`?

Reply:
Yes, let's go with the longer version to differentiate it from tagging datasets.